src/transformers/utils/quantization_config.py

```diff
@@ -33,6 +34,11 @@
 logger = logging.get_logger(__name__)
 
 
+class QuantizationMethod(Enum):
+    BITS_AND_BYTES = "bitsandbytes"
+    GPTQ = "gtpq"
+
+
 @dataclass
 class BitsAndBytesConfig:
```

Comment:
Should probably be renamed `QuantizationConfig` with an alias to `BitsAndBytesConfig` for backward compatibility.

Reply:
Maybe better to leave it as it is ? As @younesbelkada said, we were thinking of adding `AutoGPTQConfig`.

Reply:
> In this case there should be a mixin with the base methods common to both.

that sounds great !